File: BackEnd/ISA_2022/src/main/java/groupJASS/ISA_2022/DTO/RegisterNonRegisteredUserDTO.java
package groupJASS.ISA_2022.DTO;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.Valid;

@NoArgsConstructor
@Getter
@Setter
public class RegisterNonRegisteredUserDTO {
    @Valid
    private BloodUserDTO bloodUserDTO;
    @Valid
    private NonRegisteredUserInfoDTO nonRegisteredUserInfoDTO;
    @Valid
    private AddressRegUserDTO addressRegUserDTO;


}

File: FrontEnd/ISA/src/app/features/register-user/Model/RegisterNonRegisteredUserDTO.ts
import { AddressRegUserDTO } from "../Model/AddressRegUserDTO";
import { BloodDonor } from "./BloodDonor";
import { NonRegisteredUserInfoDTO } from "./NonRegisteredUserInfoDTO";
export interface RegisterNonRegisteredUserDTO{
  addressRegUserDTO:AddressRegUserDTO,
  nonRegisteredUserInfoDTO: NonRegisteredUserInfoDTO,
  bloodUserDTO: BloodDonor
}
